pkg/api/testimagestreamtagimport/v1/types.go

```diff
@@ -35,11 +35,22 @@ func (t *TestImageStreamTagImport) WithImageStreamLabels() *TestImageStreamTagIm
 	if t.Labels == nil {
 		t.Labels = map[string]string{}
 	}
-	t.Labels[LabelKeyImageStreamNamespace] = t.Spec.Namespace
-	t.Labels[LabelKeyImageStreamName] = t.Spec.Name
+	for k, v := range LabelsForImageStreamTag(t.Spec.Namespace, t.Spec.Name) {
+		t.Labels[k] = v
+	}
 	return t
 }
 
+// LabelsForImageStreamTag returns the labels by which testimagestreamtagimports
+// for a given imagestreamtag can be selected.
+func LabelsForImageStreamTag(namespace, name string) map[string]string {
```

Comment:
We have function `SanitizeLabels` in `utils/labels.go` - could we use that?

Reply:
Updated to use that